### PR TITLE
More F2LLM-v2 results

### PR DESCRIPTION
**Pull request by Geralt-Targaryen**
### Checklist

- [x] My model has a model sheet, report, or similar
- [x] My model has a reference implementation in [`mteb/models/model_implementations/`](https://github.com/embeddings-benchmark/mteb/tree/main/mteb/models/model_implementations), this can be as an API. Instruction on how to add a model can be found [here](https://embeddings-benchmark.github.io/mteb/contributing/adding_a_model/) 
  - [ ] No, but there is an existing PR ___
- [x] The results submitted are obtained using the reference implementation
- [x] My model is available, either as a publicly accessible API or publicly on e.g., Huggingface
- [x] I *solemnly swear* that for all results submitted I have not trained on the evaluation dataset including training splits. If I have, I have disclosed it clearly.

This PR includes:

1. results on Spanish and Thai benchmarks (same as https://github.com/embeddings-benchmark/results/pull/465, which has not been merged yet).
2. updated results of SynPerChatbotConvSAClassification (prompts corrected in https://github.com/embeddings-benchmark/mteb/pull/4363).
3. results on the legacy benchmarks MTEB(eng, v1), MTEB(rus, v1), MTEB(fas, v1), MTEB(jpn, v1). Prompts added in https://github.com/embeddings-benchmark/mteb/pull/4477

### CODE REVIEW

File: results/codefuse-ai__F2LLM-v2-0.6B/54b4e2dc74e01be7126d4cf5f016af6b21edc563/SynPerChatbotConvSASatisfaction.json [hunk 1]
```diff
@@ -7,133 +7,133 @@
...
+        "accuracy": 0.868531,
+        "f1": 0.855122,
+        "f1_weighted": 0.867099,
+        "precision": 0.864004,
+        "precision_weighted": 0.868515,
+        "recall": 0.849588,
+        "recall_weighted": 0.868531,
+        "ap": 0.860551,
+        "ap_weighted": 0.860551,
+        "main_score": 0.868531,
```
Comment (Samoed) [inline results/codefuse-ai__F2LLM-v2-0.6B/54b4e2dc74e01be7126d4cf5f016af6b21edc563/SynPerChatbotConvSASatisfaction.json:129]: This is very big score change. Why this happened?

Comment (Samoed) [reply results/codefuse-ai__F2LLM-v2-0.6B/54b4e2dc74e01be7126d4cf5f016af6b21edc563/SynPerChatbotConvSASatisfaction.json:129]: Is this because of https://github.com/embeddings-benchmark/mteb/pull/4477?

Comment (Geralt-Targaryen) [reply results/codefuse-ai__F2LLM-v2-0.6B/54b4e2dc74e01be7126d4cf5f016af6b21edc563/SynPerChatbotConvSASatisfaction.json:129]: No, this is because of https://github.com/embeddings-benchmark/mteb/pull/4363. Previously, the correct prompt (for sentiment analysis) was not passed to the individual tasks, and the default prompt for classification (which is for topic classification) was applied.

**Comment by KennethEnevoldsen** [comment]
There is a few of the datasets that seems especially high across models, e.g.:

SIB200Classification, MKQARetrieval, SpanishNewsClassification.v2

The pattern seems consistent across models. Can you validate that these are not leaked in the training data?

**Comment by Geralt-Targaryen** [comment]
@KennethEnevoldsen I looked into these three datasets. Both SIB200 and MKQA are included in the training data. I think the problem is that these two tasks use the training split for evaluation:

https://github.com/embeddings-benchmark/mteb/blob/f46321bafdb0f538f75017c807c1adbfa54ef50f/mteb/tasks/classification/multilingual/sib200_classification.py#L205-L217

https://github.com/embeddings-benchmark/mteb/blob/f46321bafdb0f538f75017c807c1adbfa54ef50f/mteb/tasks/retrieval/multilingual/mkqa_retrieval.py#L34-L47

SpanishNewsClassification is not in the training data. I compared the test set with 2,000 training samples from each of the training dataset, and there is not leak.

**Comment by KennethEnevoldsen** [comment]
@Geralt-Targaryen can you please update the model annotation to include the missing datasets on the training data. Note that training on the train split (even if you are only evaluating on the test split), also requires you to add the dataset to the training datasets as the model is no longer zero-shot on that specific dataset.

**Comment by KennethEnevoldsen** [comment]
> Both SIB200 and MKQA are included in the training data. I think the problem is that these two tasks use the training split for evaluation

It is not intended that you train on the training split when submitting to mteb, we seek to evaluate zero-shot evaluation. We could change it to only use the test split, but that would deprecate existing results on multiple benchmarks. I don't believe we want to do that.

**Comment by Geralt-Targaryen** [comment]
> @Geralt-Targaryen can you please update the model annotation to include the missing datasets on the training data. Note that training on the train split (even if you are only evaluating on the test split), also requires you to add the dataset to the training datasets as the model is no longer zero-shot on that specific dataset.

They are already in the training data annotation.

**Review by KennethEnevoldsen — APPROVED** [review]
rereviewed this PR - I believe it is good to merge

**Comment by Geralt-Targaryen** [comment]
Thanks. I will evaluate the models on more missing tasks these days so that the coverage test in https://github.com/embeddings-benchmark/mteb/pull/4381 can be passed.

**Comment by Geralt-Targaryen** [comment]
@KennethEnevoldsen A follow-up on the contamination: I added a note in our released training data [here](https://huggingface.co/datasets/codefuse-ai/F2LLM-v2) telling people about the contamination. Recently I took a look at other datasets, and found that these are the (text-only) tasks that use training split for evaluation:

```
BibleNLPBitextMining(name='BibleNLPBitextMining', languages=['aai', 'aak', 'aau', '...'])
NollySentiBitextMining(name='NollySentiBitextMining', languages=['eng', 'hau', 'ibo', '...'])
NusaTranslationBitextMining(name='NusaTranslationBitextMining', languages=['abs', 'bbc', 'bew', '...'])
NusaXBitextMining(name='NusaXBitextMining', languages=['ace', 'ban', 'bbc', '...'])
SIB200Classification(name='SIB200Classification', languages=['ace', 'acm', 'acq', '...'])
CodeEditSearchRetrieval(name='CodeEditSearchRetrieval', languages=['c', 'c++', 'go', '...'])
DanFeverRetrieval(name='DanFeverRetrieval', languages=['dan'])
TwitterHjerneRetrieval(name='TwitterHjerneRetrieval', languages=['dan'])
ChemHotpotQARetrieval(name='ChemHotpotQARetrieval', languages=['eng'])
NanoArguAnaRetrieval(name='NanoArguAnaRetrieval', languages=['eng'])
NanoClimateFeverRetrieval(name='NanoClimateFeverRetrieval', languages=['eng'])
NanoDBPediaRetrieval(name='NanoDBPediaRetrieval', languages=['eng'])
NanoFEVERRetrieval(name='NanoFEVERRetrieval', languages=['eng'])
NanoFiQA2018Retrieval(name='NanoFiQA2018Retrieval', languages=['eng'])
NanoHotpotQARetrieval(name='NanoHotpotQARetrieval', languages=['eng'])
NanoMSMARCORetrieval(name='NanoMSMARCORetrieval', languages=['eng'])
NanoNFCorpusRetrieval(name='NanoNFCorpusRetrieval', languages=['eng'])
NanoNQRetrieval(name='NanoNQRetrieval', languages=['eng'])
NanoQuoraRetrieval(name='NanoQuoraRetrieval', languages=['eng'])
NanoSCIDOCSRetrieval(name='NanoSCIDOCSRetrieval', languages=['eng'])
NanoSciFactRetrieval(name='NanoSciFactRetrieval', languages=['eng'])
NanoTouche2020Retrieval(name='NanoTouche2020Retrieval', languages=['eng'])
MKQARetrieval(name='MKQARetrieval', languages=['ara', 'dan', 'deu', '...'])
FaroeseSTS(name='FaroeseSTS', languages=['fao'])
```

The BEIR datasets have large-scale training splits that are probably used by many people. I think we should update the NanoBEIR benchmark. Apart from the NanoBEIR tasks, SIB200 and MKQA are the only two datasets used in training F2LLM-v2. But still I think we should update the other datasets or at least tell people about them more explicitly.
